mdns/src/conn/mod.rs

```diff
@@ -140,7 +140,7 @@ impl DnsConn {
 
     /// Close closes the mDNS Conn
     pub async fn close(&self) -> Result<()> {
-        log::info!("Closing connection");
```

Comment:
I'm not super familiar with mDNS, how often does this happen? I think having start/stop events at `info` is suitable, unless that's something that happens all the time.

Reply:
Agree! Moved that back to `info`